iset.mm

```diff
@@ -133919,6 +134256,230 @@ equal to the half of the predecessor of the odd number (which is an even
+      reexpcld cbvrabv elrab2 simprbi syl wb nn0ltexp2 syl31anc mpbird sylanbrc
+      lelttrd elnnz nnm1nn0 nncnd mullidd ltm1d peano2zm zltnle mpbid ad3antrrr
+      eqtri infssuzledc sylan2b eqbrtrid ex biimtrrid mpand mtod nltled eqbrtrd
+      1red syl2an2r crp 2rp 1zzd zsubcld rpexpcld lemuldivd cc expm1t ltdivmuld
+      2cn breqtrd df-2 breqtrdi cq nnexpcl znq mpbir2and mtbiri bitsval2 sseldd
+      flqbi elfzolt2 zlem1lt pm2.65da elfzo2 syl3anbrc ) AEJUBUCZKLDMNZOKZEUVHP
+      QZEJUVHUDNKAERUVGFUEUFAUVHALDLUHKZAUIUJZGUKULZAUVJUVHEUMQZUNZAUVNBDUMQZAU
+      VNSZUVPBTUONZDPQZUVQUVRJDUDNZKUVSUVQEUPUCZUVTUVRAUWAUVTUQUVNHURUVQUVRUWAK
+      ZLELUVRMNZUSNZUTUCZVBQZUNZUVQUWFLTVBQVAUVQUWETLVBUVQUWETVCZTUWDUMQZUWDTTV
+      DNZPQZUVQTUWCVENZEUMQUWIUVQUWLUWCEUMUVQUWCUVQUWCUVQLUVRUVKUVQUIUJUVQBUHKZ
+      UVRRKZUVQBOKZJBPQUWMAUWOUVNABAELCVFZMNZPQZCRVGZRBUWRCRVHABUWSVIPVJZUWSIAE
+      LUAVFZMNZPQZUWTUWSKUARAUWRCRVKZUXCUARVKUHRUQAUWRCUHVKZUXDVLAEVIKZLVIKZTLP
+      QZUXEAEFVMZUXGAVNUJZUXHAVOUJZELCVPVQUWRCUHRVRVSUWRUXCCUARCUAVTUWQUXBEPUWP
+      UXALMWAWBZWCWDAUXARKZUXCSZSZUWRUXAUWSCJUXOWGUWRCRUVGUEWEZUXOUXNUXAUWSKAUX
+      NWHUWRUXCCUXARUXLWIWFUXOUWPJUXAWJNKZSZEOKZUWQOKZUWRWSAUXSUXNUXQAEFWKZWLUX
+      RLOKZUWPRKZUXTWMUXQUYCUXOUWPUXAWNWOLUWPWPWQEUWQWRWTXAXBXCZXDZWKZURZUVQJDB
+      UVQXEUVQDADOKZUVNADGWKZURZXFUVQBUYGXFZUVQDADRKZUVNGURZXGUVQDBPQZUVHLBMNZP
+      QZUVQUVHEUYOUVQLDUXGUVQVNUJZUYMXIAUXFUVNUXIURZUVQUYOAUYOUHKUVNALBUVLUYEUK
+      URXHAUVNWHUVQBUWSKZEUYOPQZAUYSUVNUYDURUYSBRKZUYTELUGVFZMNZPQZUYTUGBRUWSVU
+      BBVCVUCUYOEPVUBBLMWAWBUWRVUDCUGRCUGVTUWQVUCEPUWPVUBLMWAWBXJZXKXLXMZXSAUYN
+      UYPXNZUVNAUXGUYLVUAUXHVUGUXJGUYEUXKLDBXOXPURXQZXSBXTXRZBYAXMZUKZYBYCUVQUW
+      CEUVQUWCVUKXHUYRUVQEUWCPQZBUVRUMQZUVQUVRBPQZVUMUNZUVQBUYKYDAVUNVUOXNZUVNA
+      UVROKZUWOVUPAUWOVUQUYFBYEXMUYFUVRBYFWTURYGUVQUWNVULVUMVUJUWNVULSZUVRUWSKZ
+      UVQVUMVUDVULUGUVRRUWSVUBUVRVCVUCUWCEPVUBUVRLMWAWBVUEXKZUVQVUSVUMUVQVUSSBU
+      WTUVRUMIVUSUVQVURUWTUVRUMQVUTUVQVURSZUXCUVRUWSUAJVVAWGUWSUWRCUVGVGUXCUAUV
+      GVGUXPUWRUXCCUAUVGUXLXJYIVVAVURVUSUVQVURWHVUTWFVVAUXAJUVRWJNKZSZUXSUXBOKZ
+      UXCWSAUXSUVNVURVVBUYAYHVVCUYBUXMVVDWMVVBUXMVVAUXAUVRWNWOLUXAWPWQEUXBWRWTY
+      JYKYLYMYNYOYPYQYRUVQTEUWCUVQYSUYRUVQLUVRLUUAKUVQUUBUJUVQBTUYGUVQUUCZUUDUU
+      EZUUFYGUVQUWDLUWJPUVQUWDLPQEUWCLVENZPQUVQEUYOVVGPVUFUVQLUUGKUWMUYOVVGVCUU
+      JVUILBUUHWQUUKUVQELUWCUYRUYQVVFUUIXQUULUUMUVQUWDUUNKZTOKUWHUWIUWKSXNAUXSU
+      VNUWCUHKZVVHUYAUVQUVKUWNVVIUIVUJLUVRUUOWQEUWCUUPYTVVEUWDTUVAWTUUQWBUURAUX
+      SUVNUWNUWBUWGXNUYAVUJUVREUUSYTXQUUTUVRJDUVBXMAUWOUVNUYHUVPUVSXNUYFUYJBDUV
+      CYTXQUVQUYNUVPUNZVUHAUYHUVNUWOUYNVVJXNUYIUYGDBYFYTYGUVDAUXSUVIUVJUVOXNUYA
+      UVMEUVHYFWTXQEJUVHUVEUVF $.
+  $}
+
+  ${
+    $d M m n x $.  $d N m n x $.
+    $( The bits of a number are all less than ` M ` iff the number is
+       nonnegative and less than ` 2 ^ M ` .  (Contributed by Mario Carneiro,
+       5-Sep-2016.)  (Proof shortened by AV, 1-Oct-2020.) $)
```

Comment:
```suggestion
    $( The bits of a number are all less than (2^) ` M ` iff the number is
       nonnegative and less than ` 2 ^ M ` .  (Contributed by Mario Carneiro,
       5-Sep-2016.)  (Proof shortened by AV, 1-Oct-2020.) $)
```

I'm not sure how to say this, but the original phrasing seems strange since by default bits are either 0 or 1, and the inferred meaning of 'bits' as 'numbers m such that the bit at 2^m is 1' is not noted in df-bits